site/content/recipes/explain_code_structure.md

```diff
@@ -0,0 +1,52 @@
+# Objective
+
+Reviewer has a good understanding of the overall structure of the
+code. When reviewing a section of the code, they have an idea of its
+purpose and how it relates to other parts.
+If the section to review is not yet defined, they are in a position to
+help the author choose it.
+
+# Steps
+
+1. The author describes the structure of the code to the reviewer(s).
+2. The author highlights the section to review and describes how it
+   relates to other parts of the code
+   
+# Description
+
+Even if code is isolated inside a function or module, the way it is
+written is influenced by its context. For the review to be useful, the
+reviewer(s) must have, at the very least, a rough idea of how the code
+they are looking at relates to other parts of the code. For instance,
+there could be constraints on the shape of data that comes in, or the
+data that comes out.
+
+Therefore, is it important that the author takes a few minutes to
+describe the overall structure of the code. The relationship between
```

Comment:
When and how (i.e. through what medium) would the author deliver this explanation? Should this happen via asynchronous messages? Or would it be better to merge this into one of the in-person or video chat meetings? I suppose the answer will depend on how the author and reviewer have agreed to conduct the code review, but we should give some ideas of options.

Reply:
Good point. I think this should be live, i.e. the author and reviewer sitting together looking at the code.

Reply:
sitting together = physically or virtually